static/docs/commands-reference/repro.md

```diff
@@ -67,3 +125,15 @@ Reproduce the part of the pipeline where `Posts.tsv.dvc` is the target DVC file:
     Reproducing 'Posts.tsv.dvc':
             python code/xml_to_tsv.py data/Posts.xml data/Posts.tsv python
 ```
+
+Reproduce a stage located in a subdirectory:
```

Comment:
I don't know the motivation.  Can you explain?

Reply:
To me, it is clear why you need `--cwd` for `dvc run` but not clear why it is needed for `dvc repro`. The only scenario that I see for `repro` - use default target `Dvcfile` from another dir.

It might be better to exclude `repro --cwd` from examples.

Reply:
Sure, I can remove --cwd from the dvc repro examples.  It has to be mentioned in the document because it is one of the options.

The only rationale I could concoct in my head was - a workspace with several semi-independent pipelines in subdirectories.  You could run them independently, or you could have a Dvcfile in the parent directory that can be used to run all of them.  Hence you'd run one of the subdirectory pipelines using dvc repro --cwd.

Reply:
> It has to be mentioned in the document because it is one of the options.
Absolutely.

> a workspace with several semi-independent pipelines in subdirectories
Yes, It looks like this is the only case for `repro --cwd`. If so, I don't see much value from this option.
@efiop could you please confirm this or show a scenario when `repro --cwd` bring some value.